wsmaster/che-core-api-factory-github/src/test/java/org/eclipse/che/api/factory/server/github/GithubFactoryParametersResolverTest.java

```diff
@@ -141,7 +141,7 @@ public void shouldGenerateDevfileForFactoryWithNoDevfile() throws Exception {
         .thenReturn(Optional.empty());
     Map<String, String> params = ImmutableMap.of(URL_PARAMETER_NAME, githubUrl);
     // when
-    FactoryDto factory = githubFactoryParametersResolver.createFactory(params);
+    FactoryDto factory = (FactoryDto) githubFactoryParametersResolver.createFactory(params);
```

Comment:
Cast usually is a smell of something that is not right. Can we avoid it?

Reply:
For production code I agree. But this is test code. In production code, we call `createFactory` and don't further care what's inside returned data object. We only do few more steps with it and return it from the service https://github.com/eclipse/che/blob/master/wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/FactoryService.java#L102 So we do not need to cast. Here in the test, we're testing what's really inside and I think cast in the test is ok.

WDYT? Still worth to try get rid of it?

Reply:
ok